### PR TITLE
fix syntactic error in kuberuntime_manager.go

### PR DESCRIPTION
**Pull request by qingsenLi**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubelet/kuberuntime/kuberuntime_manager.go [hunk 1]
```diff
@@ -420,7 +420,7 @@ func (m *kubeGenericRuntimeManager) podSandboxChanged(pod *v1.Pod, podStatus *ku
 	// Needs to create a new sandbox when readySandboxCount > 1 or the ready sandbox is not the latest one.
 	sandboxStatus := podStatus.SandboxStatuses[0]
 	if readySandboxCount > 1 {
-		klog.V(2).Infof("More than 1 sandboxes for pod %q are ready. Need to reconcile them", format.Pod(pod))
+		klog.V(2).Infof("More than 1 sandbox for pod %q is ready. Need to reconcile them", format.Pod(pod))
```
Comment (neolit123) [inline pkg/kubelet/kuberuntime/kuberuntime_manager.go]: i would also change this to:
> More than one sandbox is ready for Pod %q.

**Review by neolit123 — COMMENTED** [review]
thanks @qingsenLi 

added one minor nit

/ok-to-test
/priority backlog

**Comment by qingsenLi** [comment]
/cc @neolit123 
I fixed it，thanks for review

**Review by neolit123 — COMMENTED** [review]
thank you for the update.
up to the kubelet maintainers for approval at this point.

/lgtm

**Comment by qingsenLi** [comment]
/cc @yujuhong @tallclair 
thanks

**Comment by qingsenLi** [comment]
/cc @Random-Liu @vishh @yujuhong 
Needs approved label.thanks

**Comment by qingsenLi** [comment]
/assgin @tallclair 
for approval,thanks

**Comment by qingsenLi** [comment]
/assign @Random-Liu @yujuhong 
for approval,thanks

**Comment by qingsenLi** [comment]
/assign @Random-Liu @yujuhong
for approval,thanks

**Comment by lachie83** [comment]
/hold

per [release-managers discussion](https://kubernetes.slack.com/archives/CJH2GBF7Y/p1567084773002200), holding low-priority, non-milestone-targeted PRs in order to free merge pool bandwidth for 1.16-targeted PRs

feel free to unhold once there are no [lgtm+approve+green 1.16-targeted PRs](https://github.com/kubernetes/kubernetes/pulls?utf8=✓&q=is%3Apr+is%3Aopen+label%3Algtm+label%3Aapproved+milestone%3Av1.16+status%3Asuccess) remaining.

**Pull request merged**